packages/phys_shield/items/retro/vbsp_config.cfg

```diff
@@ -0,0 +1,55 @@
+"Conditions"
+	{
+	"Condition"  
+		{ 
+		"instance"	"instances/BEE2/50s/items/phys_shield/pshield_base.vmf"  
+		"result"  
+			{
+			"Pack" "PACK_FR_PLAYER_CLIP_ELECTRIC"
+			"Pack" "PACK_FR_PHYS_SHIELD_50S"
+			"Pack" "PACK_BEE2_CUST_FIZZ_MDL"
+			"Has"
+				{
+				"Fizzler"	"1"
+				"FR_pshield"	"1"
+				}
+			"CustFizzler"
+				{
+				"ModelName"		""
+				"UniqueModel"	"0"
+				"Brush"
+					{
+					"Name"		"fizz"
+					"Left"		"BEE2/fizz/phys_shield/50s_pshield_left"
+					"Center"	"BEE2/fizz/phys_shield/50s_pshield_center"
+					"Right"		"BEE2/fizz/phys_shield/50s_pshield_right"
+					"Short"		"BEE2/fizz/phys_shield/50s_pshield"
+					"Keys"
+						{
+						"classname" 	"trigger_portal_cleanser"
+						"Spawnflags" 	"4096" // Nothing
+						"visible"		"1"
+						"usescanline"	"1"
+						"StartDisabled"	"1"
+						}
+					}
+				"Brush"
+					{
+					"Name"		"clip"
+					"Left"		"BEE2/fizz/fourthreaper/player_clip_electric"
+					"Center"	"BEE2/fizz/fourthreaper/player_clip_electric"
+					"Right"		"BEE2/fizz/fourthreaper/player_clip_electric"
+					"Short"		"BEE2/fizz/fourthreaper/player_clip_electric"
+					"Nodraw"	"BEE2/fizz/fourthreaper/player_clip_electric"
+					"Keys"
+						{
+						"classname" 	"func_clip_vphysics"

```

Comment:
For clips, make it just use `tools/toolstrigger`, and remove `solidity`/`solidbsp`/`spawnflags` - those aren't used for `func_clip_vphysics`. `player_clip_electric` is only useful for player clips.